core/src/main/java/com/linecorp/armeria/client/UnprocessedRequestException.java

```diff
@@ -29,19 +29,26 @@
 
     private static final long serialVersionUID = 4679512839715213302L;
 
-    private static final UnprocessedRequestException INSTANCE = new UnprocessedRequestException(false);
-
     /**
-     * Returns an {@link UnprocessedRequestException} which may be a singleton or a new instance, depending on
-     * whether {@linkplain Flags#verboseExceptions() the verbose exception mode} is enabled.
+     * Creates a new instance with the specified {@code message} and {@code cause}.
      */
-    public static UnprocessedRequestException get() {
-        return Flags.verboseExceptions() ? new UnprocessedRequestException() : INSTANCE;
+    public UnprocessedRequestException(@Nullable String message, @Nullable Throwable cause) {
+        super(message, cause);
     }
 
-    private UnprocessedRequestException() {}
+    /**
+     * Creates a new instance with the specified {@code cause}.
+     */
+    public UnprocessedRequestException(@Nullable Throwable cause) {
+        super(cause);
```

Comment:
Set `false` to `writableStackTrace` if `Flags.verboseExceptions()` is `false`.

Reply:
I overrided `fillInStackTrace` for verbose Exception. Is it workaround?
```java
@Override
public Throwable fillInStackTrace() {
    if (Flags.verboseExceptions()) {
        super.fillInStackTrace();
    }
    return this;
}
```

Reply:
I think it's good as it is. The use of 4-arg constructor is mainly useful when we want to prevent overriding the stack trace or adding suppressed exceptions, i.e. when we want to make an exception a stateless singleton. For `UnprocessedRequestException`, that's not the case, so it's just enough to override `fillInStackTrace()`.

Reply:
@trustin Thanks for your kind explanation. 🙏